server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -841,10 +842,23 @@ exports.redirectTopologyChangeInformation = function (body, user, originator, xC
         customerJourney
       );
 
-      resolve();
+      let controlConstructUrl = onfPaths.CONTROL_CONSTRUCT;
```

Comment:
Here , we are refering the complete ControlConstruct. But , as per the response body schema , we have to ignore the profile-collection and the operation-keys.
Kindly please consider the following objects for request formulation , 
1. logical-termination-point : all instances , operation-clients/servers without operation-keys
2. forwarding-domain